core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheManager.java

```diff
@@ -614,6 +614,9 @@ private boolean restore(PageStoreDir pageStoreDir) {
       LOG.error("Failed to restore PageStore", e);
       return false;
     }
+    if (mInitService.isPresent()) {
+      mPageMetaStore.reportBlocks(pageStoreDir);
```

Comment:
Reference to this `BlockMasterSync`, The original behavior is that worker register to master with its all blocks as a full block report during the register stage.

Reply:
The changes are used to fix blocks report partly during worker register stage when we set alluxio.worker.page.store.async.restore.enabled=true (default, to accelerate worker restarts)